HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -0,0 +1,184 @@
+title:: Audio device selection
```

Comment:
not a problem to be solved here, but since lots of people use scsynth separately, it would be great if this were documented somewhere with scsynth.